WordPress/Classes/ViewRelated/Plans/PurchaseButton.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+import MRProgress
```

Comment:
Good spot. No, it's not currently. I was experimenting with using it to display a loading state for the button when a purchase is being made (like the App Store). Doesn't belong on this branch!